app/src/constants/misc.js

```diff
@@ -0,0 +1,3 @@
+const DEFAULT_ARTICLE_PREVIEW_SIZE = 200;
```

Comment:
Probably want to name this something more specific than "misc.js". Unless you're planning on just putting all constants in the same file, in which case you might just want a single "constants.js" file.

Reply:
Yeah that's true. Probably we will not be having so many constants, so it makes sense to change the file name.